### DISCUSSION TITLE
Support Discord Authentication

### DISCUSSION THREAD

**Discussion by luludotdev** [outline/outline#2335] [Ideas]
Building on #729 now that authentication is handled by Passport.js, I think it would be great to re-open the discussion surrounding supporting Discord sign-in.

Adding in a basic form of Discord auth is pretty trivial now that the stack is using Passport, however one of the biggest hurdles I've faced while experimenting locally is that there's no clear way to authorize as a specific discord guild/server and translate that into an Outline team. I'm unsure on where to go from here and I'm hoping this discussion can attract someone with a greater understanding of the project who can weigh in and offer some potential solutions to explore.

**Comment by andrewleech** [comment]
FWIW https://github.com/outline/outline/pull/2470
